mappings/remediation_advice/remediation_advice.json

```diff
@@ -1214,15 +1214,142 @@
+            },
+            {
+              "id": "source_code_dump",
+              "remediation_advice": "Obfuscate the code and find creative ways to break disassemblers and debuggers.",
+              "references": [
+                "https://en.wikipedia.org/wiki/Security_through_obscurity",
+                "https://www.researchgate.net/publication/320859156_Source_Code_Vulnerabilities_in_IoT_Software_Systems"
+              ]
+            },
+            {
+              "id": "dos_brick",
+              "remediation_advice": "Filter malicious payloads or string attacks. Apply rate limiting on the app level side.",
+              "references": [
+                "https://www.owasp.org/index.php/Application_Denial_of_Service",
+                "https://www.forbes.com/sites/leemathews/2017/04/10/a-malware-outbreak-is-bricking-insecure-iot-devices/#36603e4a29a3",
+                "https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Denial_of_Service_Cheat_Sheet.md"
+              ]
+            },
+            {
+              "id": "default_credentials",
+              "remediation_advice": "Do not ship infotainment systems with any configured accounts or with default and common usernames and passwords. Do not hard code any backdoor accounts or special access mechanisms.",
+              "references": [
+                "https://www.owasp.org/index.php/Testing_for_default_credentials_(OTG-AUTHN-002)",
+                "https://www.owasp.org/index.php/Configuration#Default_passwords"
+              ]
+            }
+          ]
         },
         {
           "id": "rf_hub",
-          "remediation_advice": ""
+          "children": [
+            {
+              "id": "key_fob_cloning",
+              "remediation_advice": "Implement key fob encryption.",
+              "references": [
+                "https://electronics.howstuffworks.com/gadgets/automotive/unlock-car-door-remote1.htm",
+                "https://www.wired.com/story/hackers-steal-tesla-model-s-seconds-key-fob/"
+              ]
+            },
+            {
+              "id": "can_injection_interaction",
+              "remediation_advice": "Implement a secure gateway that prevents CAN Injection from the RF Hub.",
+              "references": [
+                "https://www.reddit.com/r/CarHacking/comments/73qs0x/chrysler_sgw_security_gateway/"
+              ]
+            },
+            {
+              "id": "data_leakage_pull_encryption_mechanism",
+              "remediation_advice": "Inspect outgoing traffic from the RF Hub and make sure encryption mechanism cannot be extracted.",
+              "references": [
+                "https://www.networkworld.com/article/2284289/data-leak-prevention-and-encryption--tools-that-can-work-together.html"
+              ]
+            },
+            {
+              "id": "unauthorized_access_turn_on",
+              "remediation_advice": "",
+              "references": [
+                ""
```

Comment:
just make this an empty array, with no empty string inside it.

Reply:
fixed